src/Module.Server/Modes/Dtv/CrpgDtvServer.cs

```diff
@@ -191,11 +193,13 @@ private void CheckForWaveEnd()
     {
         bool viscountDead = !Mission.DefenderTeam.HasBots;
         bool defendersDepleted = Mission.DefenderTeam.ActiveAgents.Count == (viscountDead ? 0 : 1);
+        float roundDuration = _currentRoundStartTime != null ? _currentRoundStartTime.ElapsedSeconds : 0;
```

Comment:
I think MissionTime is a struct so the null check is redundant

Reply:
I've removed the null check